projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -346,6 +346,32 @@ The `EffectsModule.forFeature()` method or `provideEffects()` function must be a
 
 </div>
 
+## Standalone API in module-based apps
+
+If you have a module-based Angular application, you can still use standalone components. NgRx standalone APIs support this workflow as well.
+
+For module-based apps, you have the `EffectsModule.forRoot([...])` included in the `imports` array of your `AppModule`, which registers the root effects for dependency injection. For a standalone component with feature state/effects registered in its route configuration to successfully run effects, you will need to use the `provideEffects([...])` function in the `providers` array of your `AppModule` to register the injection token. For module-based with standalone components, you will simply have both. 
+
+<code-example header="app.module.ts">
+import { NgModule } from '@angular/core';
+import { EffectsModule } from '@ngrx/effects';
+
+import { MoviesEffects } from './effects/movies.effects';
+import * as actorsEffects from './effects/actors.effects';
+
+@NgModule({
+  imports: [
+    EffectsModule.forRoot(MoviesEffects, actorsEffects),
+  ],
+  providers: [
+    provideEffects(MoviesEffects, actorsEffects)
```

Comment:
Can you add `provideEffects` to the imports please ?

Reply:
I'm a bit confused by this request. Under the existing stand-alone section, it shows adding `provideEffects` as a provider. So I did the same here for consistency.

Reply:
Oh yeah, this is confusing, my bad 😅
I meant to add an import statement add the top of the page:

```ts
import { provideEffects } from '@ngrx/effects';
```